vnext/Microsoft.ReactNative/HStringHelper.cpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <pch.h>
+
+#include "HStringHelper.h"
+#include "Unicode.h"
+
+#include <winrt/base.h>
+
+namespace winrt {
+
+std::string HSTRINGToString(HSTRING hstring) {
```

Comment:
>std::string  [](start = 0, length = 12)

we should not need to use std::string...what are the scenarios where utf16 isn't usable?